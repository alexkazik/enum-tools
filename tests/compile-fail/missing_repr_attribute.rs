#![no_implicit_prelude]

use ::enum_tools::EnumTools;

#[derive(EnumTools, Debug, Clone, Copy, Eq, PartialEq)]
pub(crate) enum EG {
    #[enum_tools(rename = "A*")]
    A,
    C,
    B,
    D,
}

fn main() {}
