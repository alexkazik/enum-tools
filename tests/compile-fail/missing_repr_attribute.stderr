error: missing repr attribute
 --> tests/compile-fail/missing_repr_attribute.rs:6:1
  |
6 | pub(crate) enum EG {
  | ^^^
